cli/cmd/util.go

```diff
@@ -0,0 +1,88 @@
+package cmd
+
+import (
+	"crypto/rand"
+	"encoding/json"
+	"fmt"
+	"io"
+
+	"github.com/cncf/servicebroker/k8s/service_controller/model"
+)
+
+// TODO(vaikas): Move these into a ../lib or ../../lib?
+
+// from: https://play.golang.org/p/4FkNSiUDMg
+// newUUID generates a random UUID according to RFC 4122
+func NewUUID() (string, error) {
```

Comment:
github.com/satori/go.uuid is a good replacement for this. You can then just do 'uuid.NewV4().String()'